### PR TITLE
Gitpod: update cmake 

### PR DESCRIPTION
**Pull request by noraabiakar**

### CODE REVIEW

File: modcc/parser.cpp [hunk 1]
```diff
@@ -973,6 +973,14 @@ symbol_ptr Parser::parse_function() {
     auto p = parse_prototype();
     if (p == nullptr) return nullptr;
 
+    // Functions may have a unit attached
```
Comment (brenthuisman) [inline modcc/parser.cpp:976]: Is this related to the CMake update?

Comment (noraabiakar) [reply modcc/parser.cpp]: Sorry I accidentally pushed the contents of another branch. I've removed it now.

**Review by brenthuisman — APPROVED** [review]
